data-assets/acm_right_sizing_grafana_dashboard.yaml

```diff
@@ -1611,7 +1615,7 @@ data:
           {
             "allValue": null,
             "current": {
-              "selected": true,
+              "selected": false,
```

Comment:
What are we achieving by changing this to false ?

Reply:
I probably saved the dashboard at a different level of aggregation. Do we want to keep at 5 days?

Reply:
Yes by default its for 5 days only.